none

```diff
@@ -0,0 +1,11 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
What is this file (I think its a test artifact that should not be here)? If it is a test artifact, please delete it.